src/hazelcore/UHPat.re

```diff
@@ -45,6 +45,11 @@ let floatlit = (~err: ErrStatus.t=NotInHole, f: string) => FloatLit(err, f);
 
 let listnil = (~err: ErrStatus.t=NotInHole, ()) => ListNil(err);
 
+let get_var_name: operand => option(Var.t) =
```

Comment:
see comment in Action_Pat -- not sure what's going on there and why this is necessary